src/commands/api-mesh/create.js

```diff
@@ -69,6 +74,27 @@ class CreateCommand extends Command {
 			);
 		}
 
+		let filesList = [];
+
+		try {
+			filesList = getFilesInMeshConfig(data, args.file);
+		} catch (err) {
+			this.log(err.message);
+			this.error('Input mesh config is not valid.');
```

Comment:
When does this error get thrown? is it when the file is not found in the location specified? if so, please throw the error like "File not found in the location specified" which would help the user better.

Reply:
This error gets thrown during below scenarios:
1. if the files are not of js, json type
<img width="844" alt="image" src="https://user-images.githubusercontent.com/55528726/231369299-9636fb1c-2242-40f9-b7c5-e739f221d751.png">

2. if the file names are too long
<img width="941" alt="image" src="https://user-images.githubusercontent.com/55528726/231369424-fe476a76-b527-4902-bdbe-7f3442234486.png">